akka-http-cors/src/main/resources/reference.conf

```diff
@@ -0,0 +1,76 @@
+akka.http {
+
+  cors {
+
+    # If `true`, allow generic requests (that are outside the scope of the specification)
+    # to pass through the directive. Else, strict CORS filtering is applied and any
+    # invalid request will be rejected.
+    #
+    # Default: `true`
```

Comment:
Remove all the `Default: xxx` lines are they are clear from the context.
Also remove the `@see` lines, they are already in the JavaDoc.